neurom/core/_soma.py

```diff
@@ -43,24 +42,38 @@ class Soma:
     and provides iterator access to them.
     """
 
-    def __init__(self, points):
-        """Initialize a Soma object."""
-        self._points = points
+    def __init__(self, morphio_soma):
+        """Constructor.
+
+        Args:
+            morphio_soma (morphio.Soma): instance of soma of MorphIO class
+        """
+        self._morphio_soma = morphio_soma
+        # this radius is used only for `volume` method, please avoid using it for anything else.
         self.radius = 0
```

Comment:
considering the comment above, why having this as a public member ?

Reply:
Backward compatibility. NeuroM v1 should be rather questioned about this. I can make it read-only, I suppose.

Reply:
ok